lib/client.js

```diff
@@ -43,7 +43,16 @@ function Client(options) {
 
     logger = new Logger(self.options.logger);
 
-    let statsdClient = new StatsD();
+    let statsdUrl = process.env.STATSD_URL && url.parse(process.env.STATSD_URL);
```

Comment:
This isn't an ideal way to pick these up. Normally, I think we'd want to allow the library user to inject this stuff via configuration parameters.